localization/english/CWP/africa/CWP_libya_l_english.yml

```diff
@@ -0,0 +1,140 @@
+
+ gaddafi_puppet_EGY_name: "Unite with Egypt"
+ gaddafi_puppet_SDN_name: "Unite with Sudan"
+ gaddafi_puppet_PAL_name: "Unite with Palestine"
+ gaddafi_puppet_SYR_name: "Unite with Syria"
+ gaddafi_puppet_LEB_name: "Unite with Lebanon"
+ gaddafi_puppet_EOT_name: "Unite with Jordan"
+ gaddafi_puppet_TUN_name: "Unite with Tunisia"
+ gaddafi_puppet_ALD_name: "Unite with Algeria"
+ gaddafi_puppet_MOR_name: "Unite with Morocco"
+ gaddafi_puppet_ALR_name: "Unite with the Rif"
+ gaddafi_puppet_WSH_name: "Unite with Western Sahara"
+ gaddafi_puppet_MLT_name: "Unite with Malta"
+ gaddafi_puppet_MAU_name: "Unite with Mauritania"
+ gaddafi_puppet_MAL_name: "Unite with Mali"
+ gaddafi_puppet_BKF_name: "Unite with Upper Volta"
+ gaddafi_puppet_NGR_name: "Unite with Niger"
+ gaddafi_puppet_CHD_name: "Unite with Chad"
+ gaddafi_finish_unification_name: "#gold #b Centralize the Union"
+
+ gaddafi_puppet_desc: "\n#bold We receive unification points per week based on the following factors:#!\nWe have a higher global power rank: #blue 4 points#!\nWe have a stronger military: #blue 4 points#!\nWe are a major power or higher: #blue 3 points#!\nWe are neighbors: #blue 2 points#!\nWe have good relations: #blue 2 points#!\nAlways: #blue 1 point#!\nBonus points can be gained and lost through events\n\n#bold After 80 days:#!\nUnification points > #blue 150#!: We get the opportunity to diplomatically subjugate the target country\nUnification points > #blue 200#! : The target country submits to us guaranteed\n#N We have only one chance to enter into a union with this country#!"
+ gaddafi_finish_unification_desc: "\n#white We will try to annex all Jamahiriyas#!\n#red Jamahiriyas who have a higher global power rank than us can attempt to become independent#!"
+
+ je_gaddafi_unification_target: "Gaddafi's Ambitions"
+ je_gaddafi_unification_target_reason: "In the bustling heart of North Africa, a visionary leader, Muammar Gaddafi, tirelessly champions a bold and audacious dream: to unify the diverse nations of the Arab world and the vast expanse of Africa under the banner of solidarity and progress. Amidst the ever-changing geopolitical landscape, Gaddafi stands steadfast, driven by an unwavering belief in the transformative power of unity."
+ je_gaddafi_unification_points_target_desc: "#N We are in danger of becoming a Libyan Subject!#!\n\n#bold Libya receives unification points per week based on the following factors:#!\nLibya has a higher global power rank: #blue 4 points#!\nLibya has a stronger military: #blue 4 points#!\nLibya is a major power or higher: #blue 3 points#!\nWe are neighbors: #blue 2 points#!\nWe have good relations: #blue 2 points#!\nAlways: #blue 1 point#!\nBonus points can be gained and lost through events\n\n#bold After time out:#!\nUnification points > #blue 150#!: They get the opportunity to diplomatically subjugate us\nUnification points > #blue 200#!: We will submit to them guaranteed"
```

Comment:
Unlike the rest of the file, the *Journals* section is indented at 1 space rather than 2. This throws errors.

Reply:
Thanks for the tips, localization (especially in a foreign language) is probably what I am worst at
I have removed unnecessary adjectives and repetitions, if the descriptions are still too long let me know
Event 101 I have reworked to offer the player a more interesting decision